src/bridge/views/ViewPoint.js

```diff
@@ -71,7 +71,8 @@ class ViewPoint extends React.Component {
             className={'mt-8'}
             disabled={valid === false}
             onClick={() => {
-              setPointCursor(Maybe.Just(ob.patp2dec(pointName)));
+              setPointCursor(Maybe.Just(parseInt(ob.patp2dec(pointName), 10)));
```

Comment:
the latest version of urbit-ob has a `patp2bn` function that we could use `ob.patp2bn(pointName).toNumber()` if we think that's better than using `parseInt` on the string value here

Reply:
Maybe `2dec` should just give you a number like it says it'll do. (^:

But I'm fine using `2bn` until that happens.

Reply:
@Fang- looks like [2bn isn't exported](https://github.com/urbit/ob-js/blob/master/src/internal/co.js#L143) so I think this is fine to leave for now.

Reply:
(Just remembered it's not giving you a number because it supports working with numbers larger than Javascript's supported limit. Luckily, in the Bridge case, we're bounded to 32-bit numbers thanks to not caring about moons and comets.)

Reply:
hmm, in that case I'll create an issue to help us migrate to 100% bignumber, since it's sorta lucky that we won't be bitten by that for planets. It's odd to me that the azimuth-js call returns javascript numbers, which is probably why we're using the native number type everywhere else in bridge.

Reply:
> migrate to 100% bignumber

Inside Bridge? Yuck, that doesn't feel comfy at all. We don't really need bignums anywhere, do we? All web3 stuff that could need it is strings, and that's it?

> It's odd to me that the azimuth-js call returns javascript numbers

iirc most of these just return whatever came out of the under-the-hood web3 calls. Though there may be weird cases. (And in azimuth-js, too, we don't expect to be dealing with >32-bit numbers.)

Reply:
ah, are moons and comets never actually on-chain?

Reply:
Nope. Azimuth has a strictly 32-bit namespace, and doesn't have a concept of moons or comets. They're an Arvo thing, and "owned" by their planet and nobody respectively.

Reply:
sounds great, then! will ignore bignum problems 👍